activateEvents.js

```diff
@@ -6,11 +6,18 @@ var CarLot = (function (originalCarLot) {
     for (var i = 0; i < inventory.length; i++) {
       let selected = document.getElementById(`eachCar${[i]}`)
       selected.addEventListener("click", function(e) {
+        // increase border width and add background color to eachCar element
+        // reset must come first and THEN set due to CSS cascading 
+        CarLot.resetBorderBackground();
+        CarLot.setBorderBackground(selected,'cyan');
+        
         // clickedID targets carDescription0(,1 & 2) individually & gives access of carDescription to mirrorText below
         clickedID = e.target.parentElement.lastChild.id
```

Comment:
This, unfortunately, is very fragile because if the user clicks on the "Color: FireBrick", etc portion of the card, it breaks. Take a look:
![screen shot 2016-04-07 at 8 29 11 pm](https://cloud.githubusercontent.com/assets/3247835/14371811/a721791a-fd00-11e5-83af-616ede56593d.png)
Try wrapping that portion in a tag, like the card elements are, and it will fix your problem.